Code/Editor/WelcomeScreen/WelcomeScreenDialog.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QDialog>
+#endif
+
+namespace News {
+    class ResourceManifest;
+    class ArticleViewContainer;
+}
+
+namespace Ui {
+    class WelcomeScreenDialog;
+}
```

Comment:
The Style guide has the curly brace of the opening of a namespace on a newline

Reply:
nevermind that file, I had an issue with the branch. if you check the latest, this file is unchanged